src/test/java/core/basesyntax/DataConverterImplTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax;
```

Comment:
```suggestion
package core.basesyntax.converter;
```
The same for other test classes use the same structure